ios/FluentUI/Core/FluentUIFramework.swift

```diff
@@ -32,10 +35,21 @@ public extension Colors {
 public class FluentUIFramework: NSObject {
     @objc public static var bundle: Bundle { return Bundle(for: self) }
     @objc public static let resourceBundle: Bundle = {
+        #if SWIFT_PACKAGE
+        return Bundle.module
+        #else
         guard let url = bundle.resourceURL?.appendingPathComponent("FluentUIResources-ios.bundle", isDirectory: true), let bundle = Bundle(url: url) else {
             preconditionFailure("FluentUI resource bundle is not found")
         }
         return bundle
+        #endif
+    }()
+    @objc public static let colorsBundle: Bundle = {
+        #if SWIFT_PACKAGE
+        return SharedResources.colorsBundle
```

Comment:
why do we have a separate color bundle now?

Reply:
We need a separate color bundle in SPM because the colors live outside the path of iOS or macOS modules.

Basically, SPM cares a lot about file directories when defining modules. The roots of the iOS and macOS modules are under the `ios/` and `macos/` directories, respectively, and SPM does not allow you to know about anything that is not under the root of your directory. Since colors live in a different root so they can be shared (under `apple/`), that has to become a separate module.